brave/browser-laptop
# Removes PIA

Resolves #14763

## Submitter Checklist:


## Test Plan:
- make sure that PIA is not referenced any more
![image](https://user-images.githubusercontent.com/9574457/42861250-7db47386-8a5a-11e8-8b9c-d1b12dadbdbb.png)
- run browser with LEDGER_VERBOSE and make sure that requests don't go through PIA

## Reviewer Checklist:


Tests
